grammars/gfm.cson

```diff
@@ -281,11 +281,11 @@
     'name': 'comment.hr.gfm'
   }
   {
-    'begin': '^\\s*[`~]{3,}\\s*(?i:(coffee-?(script)?))\\s*$'
+    'begin': '^\\s*(([`~])\\2{2,})\\s*(?i:(coffee-?(script)?|cson))\\s*$'
```

Comment:
This is somewhat hard to understand at first glance.  May I suggest <code>`{3,}|~{3,}</code> instead?
Also, the end capture needs to be consistent (ie you can't start a code block with <code>```</code> and end it with ~~~) and have at least as many "ticks" as the begin pattern (so starting the code block with 3 ticks and ending it with 57 is fine, but starting with 4 and ending with 3 isn't).